types/ember-table/index.d.ts

```diff
@@ -0,0 +1,23 @@
+declare module 'ember-table/components/ember-table/component';
+declare module 'ember-table/components/ember-tbody/component';
+
+declare module 'table-components' {
```

Comment:
was this table-components module name chosen specifically for a certain need? I wonder if we can change the module name to `@gavant/ember-table` to match the package name, as long as it doesn't mess up anything with TS regarding its ability to find the types correctly.